pkg/cableengine/syncer/syncer.go

```diff
@@ -209,6 +210,9 @@ func (i *GatewaySyncer) generateGatewayObject() *v1.Gateway {
 				latencyInfo := i.healthCheck.GetLatencyInfo(&connection.Endpoint)
 				if latencyInfo != nil {
 					connection.Latency = latencyInfo.Spec
+					cable.RecordConnectionLatency(localEndpoint.Spec.CableName, &localEndpoint.Spec, &connection.Endpoint,
+						float64(latencyInfo.Spec.AverageRTT/uint64(time.Second)))
```

Comment:
In the case of prometheus, I think it'd be better to expose the LastRTT, in that way prometheus can be queried for average over specific times with it's query language.